shell.h

```diff
@@ -65,4 +67,28 @@ typedef struct cmd_s
  * Implemented Builtin functions
  */
 
+/*
+ * the linked list node and functions for environment variables.
+ */
+
+/**
+ * struct Node - Represents a node in a linked list.
+ * @key: The key associated with the node.
+ * @value: The value associated with the key.
+ * @next: Pointer to the next node in the linked list.
+ */
+
+typedef struct Node
+{
+	char *key;
+	char *value;
+	struct Node *next;
+} Node;
```

Comment:
can you use here a meaningful name instead of Node
- struct in C should prefixed with `s_` and typedef with `t_`, you can use for example, `struct s_env` -> `t_env`,